app/commands/infrastructure/analyze_table.rb

```diff
@@ -0,0 +1,9 @@
+class Infrastructure::AnalyzeTable
```

Comment:
Not sure if this is the right module to put this in

Reply:
Yeah, seems good!